l10n_ch_fds_postfinance/models/fds_authentication_keys.py

```diff
@@ -19,14 +19,20 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-
 from openerp import models, fields, api
 from openerp.tools.config import config
-from Crypto.PublicKey import RSA
-from Crypto import Random
+
+try:
+    from Crypto.PublicKey import RSA
+    from Crypto import Random
+except ImportError:
+    raise ImportError(
+        'This module needs Crypto to generate SSH keys. '
+        'Please install pycrypto on your system. (sudo pip install pycrypto)'
+    )
```

Comment:
Here you need to log a warning. Don't raise an error.

Use:

``` python
import logging

_logger = logging.getLogger(__name__)

try:
   ...
except ImportError:
   _logger.debug("...")
```

Otherwise having this module without this dependancy will raise an error if you add an icon for it. (or any other element in static folder.)